tests/console/yast2_vnc.pm

```diff
@@ -15,7 +15,27 @@ use base "console_yasttest";
 use testapi;
 use utils;
 use version_utils qw(is_leap is_sle);
+use registration 'add_suseconnect_product';
 
+sub open_firewall_port {
+    if (is_sle('<15') || is_leap('<15.0')) {
+        # open port in firewall if it is eanbaled and check network interfaces, check long text by send key right.
+        if (check_screen 'yast2_vnc_open_port_firewall', 30) {
```

Comment:
Please use a multi-tag assert_screen instead to prevent a 30s useless waiting time

Reply:
This part of the old code too, it is just a refactor.